src/components/toolbar/toolbar.js

```diff
@@ -122,7 +122,7 @@ function mdToolbarDirective($$rAF, $mdConstant, $mdUtil, $mdTheming, $animate) {
          *
          */
         function onChangeScrollShrink(shrinkWithScroll) {
-          var closestContent = element.parent().find('md-content');
+          var closestContent = element.parent().children('md-content');
```

Comment:
Angular's jqLite does not support `.children` with a selector - [See here](https://docs.angularjs.org/api/ng/function/angular.element)